src/styles/blocks/header.scss

```diff
@@ -0,0 +1,64 @@
+.header {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  background-color: #fff;
+  padding-inline: 50px;
+  align-items: center;
+  position: relative;
+
+  &__logo {
+    height: 40px;
+    margin: auto 0;
+
+    &:hover {
+      border: 4px;
+      border-color: #00acdc;
+    }
+  }
+}
+
+.nav {
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    align-items: center;
+  }
+
+  &__link {
+    display: flex;
+    color: $nav-text-color;
```

Comment:
Ensure that the SCSS variable $nav-text-color is defined somewhere in your SCSS files. Otherwise, this will cause an error during compilation.